api/src/routing.js

```diff
@@ -56,6 +56,10 @@ const BUILDS_DB = 'https://staging.dev.medicmobile.org/_couch/builds/'; // jshin
 const cookie = require('./services/cookie');
 const app = express();
 
+// requires content-type application/x-www-form-urlencoded header
+const formParser = bodyParser.urlencoded({ limit: '32mb', extended: false });
+// requires content-type text/plain or application/xml header
+const textParser = bodyParser.text({ limit: '32mb', type: [ 'text/plain', 'application/xml', 'text/csv' ] });
```

Comment:
I'm tempted to create a `csvParser`, unless we prefer to keep text types together for some reason.